### ISSUE TITLE
Batchnorm or dropout?

### ISSUE THREAD

**Issue by fanghgit** [stepankonev/waymo-motion-prediction-challenge-2022-multipath-plus-plus#13]
Thanks for sharing your code, really nice work! I am trying to reproduce your result, I wonder if you have turned on batchnorm or dropout for your training?

**Comment by stepankonev** [comment]
Hello, you may check the configuration in `code/configs`
